client/src/components/app/App.js

```diff
@@ -1,34 +1,61 @@
 import "./App.css";
 import { Switch, Route, NavLink } from "react-router-dom";
+import axios from "axios";
+import { useState, useEffect } from "react";
 import AppHeader from "./AppHeader.js";
-import HomePage from "../../pages/HomePage.js";
-// import AboutPage from "../../pages/AboutPage.js";
-import FavsPage from "../../pages/FavsPage.js";
-import BooksPage from "../../pages/BooksPage.js";
-import BookPage from "../../pages/BookPage.js";
+
+import MyCommunitiesPage from "../../pages/MyCommunitiesPage.js";
+
+import CommunitiesContext from "../../contexts/CommunitiesContext";
+
+import Modal from "../modal/Modal.js";
+import NewCommunityModalPage from "../../pages/NewCommunityModalPage.js";
 
 function App() {
+  const [resources, setResources] = useState([]);
+
+  const primaryAddActionFormId = "newCommunity";
```

Comment:
Is this `newCommunity` value a value that never changes? Then I would recommend to make it visibile for others developers with
```js
const PRIMARY_ADD_ACTION_FORM_ID = "newCommunity";
// or
const PRIMARY_ID = "newCommunity";
```
You get the idea.

Reply:
si senor!